modules/javafx.graphics/src/main/java/javafx/scene/AccessibleAttribute.java

```diff
@@ -800,4 +821,27 @@ public enum AccessibleAttribute {
     public Class<?> getReturnType() {
         return returnClass;
     }
+
+    /**
+     * This enum describes the values for TOGGLE_STATE attribute.
+     *
+     * @see TOGGLE_STATE
+     * @since 21
+     */
+    public enum ToggleState {
+        /**
+         * Indicates that the toggle control is not selected.
+         */
+        UNCHECKED,
+
+        /**
+         * Indicates that the toggle control is selected.
+         */
+        CHECKED,
+
+        /**
+         * Indicates that the toggle state of the control cannot be determined.
+         */
+        INDETERMINATE
+    }
```

Comment:
is this the right place for ToggleState?
looks like the right place, just want to confirm.

Reply:
Yes, this is where I would recommend putting it.